ci_framework/roles/switch_config/molecule/default/converge.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  tasks:
+    - name: Do not do anything
+      ansible.builtin.debug:
+        msg: |
+          This role exclusively relies on running physical switches.
+          Molecule cannot run and mimic it in an environment without a switch to test on.
```

Comment:
We may consider adding some "intelligence" to the way we generate the molecule job list + project, but for now, it is expected we have a `molecule` job. The balance is in favor of molecule tests so far. If anyone has time to spend on the job generation script[1], patches are welcome :). Maybe adding support for a "non-molecule" list where we explicitly exclude molecule for selected roles ?

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/scripts/create_role_molecule.py

Reply:
Not sure I was asked to create this file in this format and so I did, I do agree it doesn't help in testing the role itself but I guess it is more for keeping some uniformity

Reply:
ok thanks for clarification. so this can be removed after https://github.com/openstack-k8s-operators/ci-framework/pull/929

going to mark it resolved